go.mod

```diff
@@ -3,37 +3,37 @@ module github.com/kong/kubernetes-ingress-controller
 go 1.13
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.6.0 // indirect
-	contrib.go.opencensus.io/exporter/prometheus v0.1.0 // indirect
-	contrib.go.opencensus.io/exporter/stackdriver v0.12.9 // indirect
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/eapache/channels v1.1.0
 	github.com/fatih/color v1.9.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/google/go-containerregistry v0.0.0-20200131185320-aec8da010de2 // indirect
 	github.com/hashicorp/go-memdb v1.2.0 // indirect
 	github.com/hashicorp/go-uuid v1.0.1
 	github.com/kong/deck v1.2.1
 	github.com/kong/go-kong v0.13.0
 	github.com/lithammer/dedent v1.1.0
-	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
-	github.com/mitchellh/mapstructure v1.2.2
-	github.com/openzipkin/zipkin-go v0.2.2 // indirect
-	github.com/prometheus/client_golang v1.0.0
-	github.com/sirupsen/logrus v1.4.2
+	github.com/mitchellh/mapstructure v1.3.1
+	github.com/prometheus/client_golang v1.7.1
+	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.6.2
-	github.com/stretchr/testify v1.4.0
+	github.com/spf13/viper v1.7.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
Please don't update unrelated deps with this change.

Reply:
Those have been pulled in by the updated knative (the fork is based off master to mimic the next release as closely as possible). Should I force them to stay at original versions?

Reply:
No, that's fine then. This is the part of go.mod I don't like.